### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by bigmosi**
##  In this milestone we did the following

- Implement only a basic UI of the Awesome Book project with plain HTML:
- Created an array that keeps a list of books.
- Created a function that adds a new book to the collection, with title and author.
- Created a function that removes a book from the collection
- Displaying all books saved in the collection in the top part of the page.
- Used JavaScript Array of Objects
- Add the book array to localStorage

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    });
+    localStorage.setItem('BOOKS_LIST', JSON.stringify(this.BookObj));
+  }
+
+  remove(element) {
+    const { id } = element.dataset;
+    element.parentElement.remove();
+    this.BookObj.splice(this.BookObj.findIndex((item) => item.id === parseInt(id, 10)), 1);
+    localStorage.setItem('BOOKS_LIST', JSON.stringify(this.BookObj));
+  }
```
Comment (AnselemOdims) [inline script.js]: - [ ] Good job working on this but at the moment when I add books and start removing them from the top, it removes the wrong book from the storage. Please kindly ensure that removing an item from the DOM actually removes the right item in the storage.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @bigmosi, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: script.js (hunk 1 above)
Comment (ndikumanaisaie) [reply script.js]: Thank you for your review! We fixed the problem and follow all the project requirements.

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a 

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**